dev-libs/icu/icu71-71.1.recipe

```diff
@@ -39,52 +39,51 @@ providing very competitive performance.
 a range of text, or identify locations that would be suitable for line \
 wrapping when displaying the text."
 HOMEPAGE="http://www.icu-project.org"
-COPYRIGHT="1995-2020 IBM Corporation and others."
+COPYRIGHT="1991-2022 Unicode, Inc. and others."
 LICENSE="ICU"
 REVISION="1"
-SOURCE_URI="https://github.com/unicode-org/icu/releases/download/release-70-1/icu4c-70_1-src.tgz"
-SOURCE_URI_2="https://github.com/unicode-org/icu/releases/download/release-70-1/icu4c-70_1-data.zip#noarchive"
-#SOURCE_URI_3="http://www.iana.org/time-zones/repository/releases/tzdata2019c.tar.gz#noarchive"
-CHECKSUM_SHA256="8d205428c17bf13bb535300669ed28b338a157b1c01ae66d31d0d3e2d47c3fd5"
-CHECKSUM_SHA256_2="c72723ddba3300ffb231d6b09e2a728ea6e89de10ed5927f74bacbd77042336e"
+SOURCE_URI="https://github.com/unicode-org/icu/releases/download/release-71-1/icu4c-71_1-src.tgz"
+SOURCE_URI_2="https://github.com/unicode-org/icu/releases/download/release-71-1/icu4c-71_1-data.zip#noarchive"
+#SOURCE_URI_3="http://www.iana.org/time-zones/repository/releases/tzdata2022a.tar.gz#noarchive"
+CHECKSUM_SHA256="67a7e6e51f61faf1306b6935333e13b2c48abd8da6d2f46ce6adca24b1e21ebf"
+CHECKSUM_SHA256_2="e3882b4fece6e5e039f22c3189b7ba224180fd26fdbfa9db284617455b93e804"
 SOURCE_DIR="icu"
-PATCHES="icu70-70.1.patchset"
+PATCHES="icu71-71.1.patchset"
 
-ARCHITECTURES="!all ?x86 ?arm ?sparc ?arm64 ?riscv64 x86_64"
-# leave inactive until full testing can be done
```

Comment:
Did you tested it on riscv?

Reply:
No. Issues mentioned on Haiku boot. See: https://cgit.haiku-os.org/haiku/commit/?id=ca49b33f40b720d2b52a69ed5f3b00117b3b352b